storage/cloud-client/requester_pays_test.py

```diff
@@ -23,7 +23,14 @@
 import storage_download_file_requester_pays
 import storage_get_requester_pays_status
 
-BUCKET = os.environ["CLOUD_STORAGE_BUCKET"]
+
+# We should not use the same bucket as other tests are using.
+# First look at REQUESTER_PAYS_TEST_BUCKET and fall back to
+# ${CLOUD_STORAGE_BUCKET} + '-requester-pays-test'.
+BUCKET = os.environ.get(
+    "REQUESTER_PAYS_TEST_BUCKET",
```

Comment:
Should this be added to https://github.com/GoogleCloudPlatform/python-docs-samples/blob/bb83cea15029f50e8f82c6c2d50c971b35b48c33/testing/test-env.tmpl.sh?

Reply:
Good catch, I'll add